src/main/kotlin/org/jdc/template/datasource/webservice/individuals/IndividualService.kt

```diff
@@ -0,0 +1,17 @@
+package org.jdc.template.datasource.webservice.individuals
+
+import org.jdc.template.datasource.webservice.individuals.dto.DtoIndividuals
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface IndividualService {
+
+    @GET("/mobile/interview/directory")
+    fun individuals(): Call<DtoIndividuals>
+
+    companion object {
+        const val BASE_URL = "https://ldscdn.org"
+        const val SUB_URL = "/mobile/interview/directory"
```

Comment:
Try to keep things DRY (Don't Repeat Yourself). In this case you can reuse SUB_URL for @GET argument